streams/src/test/java/org/apache/kafka/streams/processor/internals/TasksTest.java

```diff
@@ -51,69 +39,16 @@ public class TasksTest {
-        final Map<TaskId, Set<TopicPartition>> activeTasks = mkMap(
-            mkEntry(statefulTask.id(), statefulTask.changelogPartitions()),
-            mkEntry(statelessTask.id(), statelessTask.changelogPartitions())
-        );
-        final Map<TaskId, Set<TopicPartition>> standbyTasks =
-            mkMap(mkEntry(standbyTask.id(), standbyTask.changelogPartitions()));
-        when(activeTaskCreator.createTasks(mainConsumer, activeTasks)).thenReturn(Arrays.asList(statefulTask, statelessTask));
-        when(standbyTaskCreator.createTasks(standbyTasks)).thenReturn(Collections.singletonList(standbyTask));
-
-        tasks.createTasks(activeTasks, standbyTasks);
-
-        final Exception exceptionForStatefulTaskOnTask = assertThrows(IllegalStateException.class, () -> tasks.task(statefulTask.id()));
-        assertEquals("Task unknown: " + statefulTask.id(), exceptionForStatefulTaskOnTask.getMessage());
-        assertFalse(tasks.activeTasks().contains(statefulTask));
-        assertFalse(tasks.allTasks().contains(statefulTask));
-        final Exception exceptionForStatefulTaskOnTasks = assertThrows(IllegalStateException.class, () -> tasks.tasks(mkSet(statefulTask.id())));
-        assertEquals("Task unknown: " + statefulTask.id(), exceptionForStatefulTaskOnTasks.getMessage());
-        final Exception exceptionForStatelessTaskOnTask = assertThrows(IllegalStateException.class, () -> tasks.task(statelessTask.id()));
-        assertEquals("Task unknown: " + statelessTask.id(), exceptionForStatelessTaskOnTask.getMessage());
-        assertFalse(tasks.activeTasks().contains(statelessTask));
-        assertFalse(tasks.allTasks().contains(statelessTask));
-        final Exception exceptionForStatelessTaskOnTasks = assertThrows(IllegalStateException.class, () -> tasks.tasks(mkSet(statelessTask.id())));
-        assertEquals("Task unknown: " + statelessTask.id(), exceptionForStatelessTaskOnTasks.getMessage());
-        final Exception exceptionForStandbyTaskOnTask = assertThrows(IllegalStateException.class, () -> tasks.task(standbyTask.id()));
-        assertEquals("Task unknown: " + standbyTask.id(), exceptionForStandbyTaskOnTask.getMessage());
-        assertFalse(tasks.allTasks().contains(standbyTask));
-        final Exception exceptionForStandByTaskOnTasks = assertThrows(IllegalStateException.class, () -> tasks.tasks(mkSet(standbyTask.id())));
-        assertEquals("Task unknown: " + standbyTask.id(), exceptionForStandByTaskOnTasks.getMessage());
-        verify(activeTaskCreator).createTasks(mainConsumer, activeTasks);
-        verify(standbyTaskCreator).createTasks(standbyTasks);
-        verify(stateUpdater).add(statefulTask);
-    }
 
     @Test
-    public void shouldCreateTasksWithoutStateUpdater() {
-        final Tasks tasks = new Tasks(logContext, activeTaskCreator, standbyTaskCreator, null);
-        tasks.setMainConsumer(mainConsumer);
+    public void shouldCreateTasks() {
```

Comment:
The names of the tests here and in `StandbyTaskTest` and `StreamTaskTest` do not reflect the content/renamed methods anymore.

Reply:
ack.